Server/app.ts

```diff
@@ -1,23 +1,35 @@
 import express, { Request, Response } from 'express';
 import dotenv from 'dotenv';
 import bodyParser from 'body-parser';
-import appRouter from './routes';
-import errorHandler from './middlewares';
+import appRouter from './router';
+import { applicationErrorHandler, userErrorHandler } from './middlewares/errors/errorHandler';
+import morgan from 'morgan';
```

Comment:
When importing, the order of imports should be as follow:
1. external imports (mongoose, morgan, express)
2. internal imports